app/build.gradle.kts

```
@@ -0,0 +1,147 @@
+import com.android.build.gradle.internal.tasks.factory.dependsOn
+import com.github.spotbugs.snom.SpotBugsTask
+
+plugins {
+    id("com.android.application")
+    id("com.github.spotbugs")
+}
+
+spotbugs {
+    ignoreFailures.set(false)
+    setEffort("max")
+    excludeFilter.set(file("./config/spotbugs/exclude.xml"))
+    reportsDir.set(layout.projectDirectory.file("build/reports/spotbugs/").asFile)
```

Any reason not to use `buildDirectory` here?

```suggestion
    reportsDir.set(layout.buildDirectory.file("reports/spotbugs/").get().asFile)
```

(not sure why it's a property, thus needing `.get()`, though)

---

Done, thanks.